app/routes/ui/admin_scene.py

```diff
@@ -33,15 +33,17 @@ async def get(self, scene_id):
 
             await self.render(
                 "scene.html",
-                is_last_scene=False,
+                is_last_scene="false",
```

Comment:
Any reason for string over the actual boolean value?

Reply:
Javascript vs Python boolean. `False` is an undefined variable in HTML/JS, whereas "false" works